kinto_wizard/__main__.py

```diff
@@ -55,7 +58,8 @@ def main():
 
     thread_pool = ThreadPoolExecutor()
     event_loop = asyncio.get_event_loop()
-    async_client = AsyncKintoClient(client, event_loop, thread_pool)
+    async_client = AsyncKintoClient(client, event_loop, thread_pool,
+                                    dry_run=getattr(args, 'dry_run', False))
```

Comment:
wouldn't it be nice to set a default at the subparser level ?

Reply:
Actually it didn't make sense to add a --dry-run for the dump command, and we have a default for the load command.